spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/src/main/java/org/springframework/cloud/gcp/core/autoconfig/GoogleCloudContextAutoConfiguration.java

```diff
@@ -14,26 +17,26 @@
 
 /**
  * @author Vinicius Carvalho
+ *
+ * Base starter for Google Cloud Projects. Provide defaults for {@link GoogleCredentials}.
+ * Binds properties from {@link org.springframework.cloud.gcp.core.GoogleCloudProperties}
+ *
  */
 @Configuration
 @ConditionalOnClass(GoogleCredentials.class)
-public class CredentialsAutoConfiguration implements EnvironmentAware {
+@EnableConfigurationProperties(GoogleCloudProperties.class)
+public class GoogleCloudContextAutoConfiguration {
```

Comment:
I'm just afraid that using Gcp in some places and GoogleCloud in others can be a little confusing. I myself got confused with Spring-related naming before, so maybe it's not that small deal.
In any case, I too don't want to waste anyone's time over this. It seems GoogleCloud is the consensual choice, so be it.